drivers/ethernet/eth_enc424j600.c

```diff
@@ -684,6 +686,8 @@ static int enc424j600_init(const struct device *dev)
 		return -EIO;
 	}
 
+	enc424j600_write_sbc(context->dev, ENC424J600_1BC_CLREIE);
```

Comment:
Actually it is simpler to change 
```
enc424j600_set_sfru(dev, ENC424J600_SFR3_EIEL,
      			    ENC424J600_EIE_PKTIE | ENC424J600_EIE_LINKIE);
```

to 
```
enc424j600_write_sfru(dev, ENC424J600_SFR3_EIEL,
      			    ENC424J600_EIE_PKTIE | ENC424J600_EIE_LINKIE);
```

and place it in this line. This is what I wrote in the email as an example, all this is a bit too exhausting, I will open an alternative PR.